packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -841,6 +841,7 @@ class EditableText extends StatefulWidget {
     this.onAppPrivateCommand,
     this.onSelectionChanged,
     this.onSelectionHandleTapped,
+    this.groupId,
```

Comment:
Should we also add this to the `CupertinoTextField` and related widgets in the Cupertino library?

Reply:
Yes, thank you for your reminder